src/app/form-builder/components/dialogs/preview-dialog/preview-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<h2 mat-dialog-title>Preview: {{data.formName}}</h2>
+<form class="form">
+  <div class="field"
+       *ngFor="let field of data.fields; let i = index"
+       [ngSwitch]="field.type"
+  >
+    <div *ngSwitchCase="'text'">
+      <label>{{field.availableStyles.placeholder || 'Text'}}</label>
+      <input type="text" class="form-control" [ngStyle]="field.availableStyles"
+             [required]="field.availableStyles.required" [placeholder]="field.availableStyles.placeholder || 'Text'">
+    </div>
+    <div *ngSwitchCase="'textarea'">
```

Comment:
create enum for strings: textarea, button, checkbox etc...
```suggestion
    <div *ngSwitchCase="'textarea'">
```